examples/ec2/vpcendpointpolicy.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: VPCEndpointPolicy
+metadata:
+  annotations:
+    meta.upbound.io/example-id: ec2/v1beta1/vpcendpointpolicy
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    policy: |-
+      ${jsonencode({
+          "Version" : "2012-10-17",
+          "Statement" : [
+            {
+              "Sid" : "AllowAll",
+              "Effect" : "Allow",
+              "Principal" : {
+                "AWS" : "*"
+              },
+              "Action" : [
+                "dynamodb:*"
+              ],
+              "Resource" : "*"
+            }
+          ]
+        })}
+    region: us-west-1
+    vpcEndpointIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+
+---
+
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: VPC
+metadata:
+  annotations:
+    meta.upbound.io/example-id: ec2/v1beta1/vpcendpointpolicy
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    cidrBlock: 10.0.0.0/16
+    region: us-west-1
+
+---
+
+apiVersion: ec2.aws.upbound.io/v1beta1
+kind: VPCEndpoint
+metadata:
+  annotations:
+    meta.upbound.io/example-id: ec2/v1beta1/vpcendpointpolicy
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    region: us-west-1
+    serviceNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: aws_vpc_endpoint_service
+    vpcIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+
+---
```

Comment:
Could you please remove this `---`? This statement at the end of the file causes the end-to-end tests to fail.